typing/env.ml

```diff
@@ -1655,29 +1669,40 @@ let rec components_of_module_maker
             let final_decl = Subst.type_declaration prefixing_sub fresh_decl in
             Btype.set_row_name final_decl
               (Subst.type_path prefixing_sub (Path.Pident id));
-            let constructors =
-              List.map snd
-                (Datarepr.constructors_of_type ~current_unit:(get_unit_name ())
-                   path final_decl)
+            let descrs = match decl.type_kind with
+            | Type_variant _ ->
```

Comment:
I don't like this non-indentation style, and I don't think it's used elsewhere in the codebase, certainly not in this file.

Reply:
Would
```ocaml
let descrs = match decl.type_kind with
  | Type_variant _ ->
```
be preferable ?
Or rather
```ocaml
let descrs =
  match decl.type_kind with
  | Type_variant _ ->
```

Reply:
I have a personal reference for the second, but either is fine honestly (I believe both are used in the codebase, with the second being perhaps more popular in this particular file)